BIPs/BIP-162/ethereumTreasury.json

```diff
@@ -0,0 +1,151 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1674765376433,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.11.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x2c1a21c02001b11403a042fcd0e2c7958880db1f00b910e2a8dd7a7e8979fd95"
+  },
+  "transactions": [
+    {
+      "to": "0x7B50775383d3D6f0215A8F290f2C9e2eEBBEceb2",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "address", "name": "recipient", "type": "address" },
+          { "internalType": "uint256", "name": "amount", "type": "uint256" }
+        ],
+        "name": "transfer",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "recipient": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+        "amount": "682682196629189057534951"
+      }
+    },
+    {
+      "to": "0xA6F548DF93de924d73be7D25dC02554c6bD66dB5",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "address", "name": "recipient", "type": "address" },
+          { "internalType": "uint256", "name": "amount", "type": "uint256" }
+        ],
+        "name": "transfer",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "recipient": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+        "amount": "58598401025232390138"
+      }
+    },
+    {
+      "to": "0xfF083f57A556bfB3BBe46Ea1B4Fa154b2b1FBe88",
```

Comment:
WETH/GTC BPT (contract 0xfF083f57A556bfB3BBe46Ea1B4Fa154b2b1FBe88) amount: 7725430893893258147597 Note: GTC will be returned to the DAO Treasury.